docs/your-questions-answered.md

```diff
@@ -0,0 +1,13 @@
+# Your Questions Answered
+
+This section of the Cloud Service Certification wiki answers questions asked by the CSC community.
+
+Ask your question by opening a [Support Issue](https://github.com/finos/cloud-service-certification/issues/new?template=Support_question.md) through GitHub.
+
+## Question
+How does Cloud Service Certification maintain the security of the environment and the protection of the environment for open source consumers? #18
```

Comment:
Should we try to include the reference link to the question here? e.g. making the number the link.

Reply:
Great suggestion. Thanks :)